python/requirements_test.txt

```diff
@@ -53,7 +53,8 @@ PyOpenSSL==22.1.0
 pygame==2.1.2
 Pygments==2.13.0
 pymongo==4.3.2
-pyspark==3.3.1
+# TODO: Replace this with pyspark==3.4 once it is released.
+https://ml-team-public-read.s3.us-west-2.amazonaws.com/pyspark-3.4.0.dev0.tar.gz
```

Comment:
Note:

Spark stage scheduling feature (on standalone mode spark cluster) is introduced in apache/spark 3.4 (it will be released this month)

So for testing purpose, I built a package from apache/spark master